src/components/account/details/FormField.js

```diff
@@ -4,12 +4,21 @@ import '../../../styles/account/details/form_field.css'
 
 export default class EditAccountDetails extends Component {
   render() {
-    const { label, value } = this.props
-    return (
-      <div className="form--read-only">
-        <div className="form--label">{ label }</div>
-        <div className="form--field">{ value }</div>
-      </div>
-    )
+    const { label, value, editing } = this.props
+    if ( !editing ) {
```

Comment:
Please separate setup from logic with a blank line.  As a matter of style, I prefer to start with the "truthy" case in an if/else statement, if possible - it reduces the cognitive load of understanding what's happening (I don't have to mentally invert a boolean expression).

Finally, this block of code has a lot of repetition, when the only thing that varies between the two is the second element in the containing div.  I suggest refactoring to enclose only the part that varies in the `if/else`.